mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -1157,19 +1161,19 @@ msgid "cherry_found"
 msgstr "Vous avez trouvé une cerise."
 
 msgid "xero_grunts_block"
-msgstr "This area is under construction, it should be done soon."
+msgstr "Cette zone est encore en travaux, elle devrait être prête bientôt."
 
 msgid "waiting_for_misa"
-msgstr "Our leader would like to talk with you. When you are ready, talk to my friend next to me."
+msgstr "Notre chef voudrait vous parler. Quand vous êtes disponibles, parlez à mon ami juste à côté de moi."
```

Comment:
No 's' at the end of "disponible", because it's singular vous.

https://grammaire.reverso.net/nous-vous-accord-au-singulier/

Reply:
...whoops